config.json

```diff
@@ -13,6 +13,7 @@
     "device": ""
   },
   "remove_cancelled_classes": true,
+  "location_means_cancelled": [""],
```

Comment:
Can you phrase `location_means_cancelled` differently? And perhaps set it to boolean `false` by default, your feature should only be run if it's is enabled.